test-case/src/WarnXdebugPhpunitExtension.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\TestCase;
+
+use PHPUnit\Runner\BeforeFirstTestHook;
+
+class WarnXdebugPhpunitExtension implements BeforeFirstTestHook
+{
+    public function executeBeforeFirstTest(): void
+    {
+        if (\is_callable('xdebug_info') && [] !== xdebug_info('mode') && xdebug_info('mode') !== ['off']) {
```

Comment:
According to https://php.watch/articles/xdebug_info, the return value of `xdebug_info()` is a string. 🤔

Reply:
Could this be the case for Xdebug 2? At least for me with version 3 it's an array.

Reply:
It is `mixed`: https://xdebug.org/docs/all_functions#xdebug_info and an array for `'mode'`